src/main/java/com/laserfiche/repository/api/clients/impl/ApiClientUtils.java

```diff
@@ -89,6 +89,12 @@ protected static ApiException createApiException(HttpResponse<Object> httpRespon
         return ApiException.create(statusCode, headersMap, problemDetails, null);
     }
 
+    protected static ApiException createApiExceptionForExportDocumentAsStream(HttpResponse<byte[]> httpResponse, ProblemDetails problemDetails) {
```

Comment:
This function looks the same as the above `createApiException` with the only difference being httpResponse has type `HttpResponse<byte[]>`. Maybe instead you could update `createApiException` with parameter `HttpResponse<?>` so that it can be used by the `exportDocumentAsStream` function

Reply:
ok done, thanks!